plugin/keyring_vault/keyring_vault-test/vault_keys_container_ex.h

```diff
@@ -26,7 +26,7 @@ namespace keyring {
 
 class Vault_keys_container_ex : public Vault_keys_container {
  public:
-  Vault_keys_container_ex(ILogger *logger) : Vault_keys_container(logger) {}
+  Vault_keys_container_ex(ILogger *_logger) : Vault_keys_container(_logger) {}
```

Comment:
Not probably significantly important but leading underscores in identifier names are are reserved for standard library.